.github/workflows/R-CMD-check.yml

```diff
@@ -33,13 +33,16 @@ jobs:
       R_REMOTES_NO_ERRORS_FROM_WARNINGS: true
       GITHUB_PAT: ${{ secrets.GITHUB_TOKEN }}
       R_REPOS: ${{ matrix.config.repos }}
+      R_KEEP_PKG_SOURCE: yes
```

Comment:
I'm not a yaml wizard but are `yes` and `true` both valid boolean values? I'd assume so and for consistency with the existing booleans I'd stick to `true`.